modules/common/src/main/MarkdownRender.scala

```diff
@@ -262,6 +271,12 @@ object MarkdownRender:
         .tag("/div")
         .unit
 
+    //Catches the site headers for https://github.com/lichess-org/lila/issues/11450
+    private def matchesSiteHeader(node : LinkNode) : Boolean = {
+      val parentValue : BasedSequence = Option(node.getParent()).get.getChars()
```

Comment:
We'll change this as well, because if getParent returns null then Option.get will throw an exception.